src/modules/item/Item.tsx

```diff
@@ -329,11 +328,10 @@ const ItemContentWrapper = ({ item }: { item: ItemRecord }) => {
   const { data: itemTags } = useItemTags(item.id);
   const isItemHidden = isHidden(item, itemTags);
 
-  return (
-    <HiddenWrapper itemId={item.id} hidden={isItemHidden}>
-      <ItemContent item={item} />
-    </HiddenWrapper>
-  );
+  if (isItemHidden) {
```

Comment:
Maybe let's add a comment : "An item the user has access to can be hidden (write, admin) so we hide it in player"